common/oatbox/filesystem/FileSystemHandler.php

```diff
@@ -22,13 +22,18 @@
 
 namespace oat\oatbox\filesystem;
 
+use oat\oatbox\service\ServiceManager;
+use ReflectionClass;
+use ReflectionProperty;
 use Zend\ServiceManager\ServiceLocatorAwareInterface;
 use Zend\ServiceManager\ServiceLocatorAwareTrait;
 
 abstract class FileSystemHandler implements ServiceLocatorAwareInterface
 {
     use ServiceLocatorAwareTrait;
 
+    private const UNSERIALIZABLE_PROPERTIES = ['fileSystem', 'serviceLocator'];
```

Comment:
I just realized that `unserializable` sounds quite ambiguous.
Let's change it to `NOT_SERIALIZABLE` please :)